README.md

```diff
@@ -11,3 +11,33 @@ For more information about how the admin container fits into the Bottlerocket op
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Authenticating with the Admin Container
+
+Starting from v0.6.0, users have the option to pass in their own ssh keys rather than the admin container relying on the AWS instance metadata service (IMDS).
+
+Users can add their own keys by populating the admin container's user-data with a base64-encoded JSON block.
+(If you would like to continue using the public keys available in IMDS this is already done for you!)
+
+To use custom public keys to .ssh/authorized_keys you will want to generate a JSON-structure like this:
+
+```
+{
+   "ssh":{
+      "authorized_keys":[
+         "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQClKsfkNkuSevGj3eYhCe53pcjqP3maAhDFcvBS7O6V
+          hz2ItxCih+PnDSUaw+WNQn/mZphTk/a/gU8jEzoOWbkM4yxyb/wB96xbiFveSFJuOp/d6RJhJOI0iBXr
+          lsLnBItntckiJ7FbtxJMXLvvwJryDUilBMTjYtwB+QhYXUMOzce5Pjz5/i8SeJtjnV3iAoG/cQk+0FzZ
+          qaeJAAHco+CY/5WrUBkrHmFJr6HcXkvJdWPkYQS3xqC0+FmUZofz221CBt5IMucxXPkX4rWi+z7wB3Rb
+          BQoQzd8v7yeb7OzlPnWOyN0qFU0XA246RA8QFYiCNYwI3f05p6KLxEXAMPLE my-key-pair"
```

Comment:
I think the embedded newlines may confuse some users - can we show this as it "should" be provided? As a single line?